lib/collections/users.js

```diff
@@ -288,6 +289,20 @@ Meteor.methods({
     }
   },
 
+    'user.message'(email, message) {
```

Comment:
Two things. One is a potential for abuse that I hadn't thought of before (or forgot about), which is that a single user could spam out tons of emails this way if they were malicious. Could we set a limit - say, 10 - per user, and each time they successfully send a message via this system we ensure they haven't sent more than that number of emails? This could help prevent theoretical abuse to players and the email server.

Second question, much more nitpicky: could we change `user.message` to be slightly more descriptive, eg `user.matchmaking-message`? I prefer the explicit name and it'll be easier to chase down in the future.

Reply:
Changed the name.

Rate limiting the messages is a good idea, but I don't have bandwidth to implement it in the near future.  I'll create an issue to track this.